include/psa/crypto_extra.h

```diff
@@ -2075,14 +2075,33 @@ typedef enum psa_crypto_driver_pake_step {
     PSA_JPAKE_X4S_STEP_ZK_PROOF   = 12  /* Round 2: input Schnorr NIZKP proof for the X4S key (from peer) */
 } psa_crypto_driver_pake_step_t;
 
+typedef enum psa_jpake_round {
```

Comment:
Is it okay if I just remove the documentation in `psa_crypto_pake.c`? It feels odd to have the implementation referencing the internal details of the dispatch layer.

Reply:
(To clarify, I've assumed that you mean [this comment](https://github.com/Mbed-TLS/mbedtls/blob/development/library/psa_crypto_pake.c#L82))

Reply:
It seems `check_names.py` has picked this up as well.

Reply:
There are two large comments at the beginning of `psa_crypto_pake.c`. I'd say the first one can be kept as a reminder of the sequence of calls to the PAKE APIs for J-PAKE. The second one is not relevant anymore. It would probably be nice to replace it with the possible sequence of calls to `mbedtls_psa_pake_input/output()` with the value of the `step`.